test/opset_pass/product_opset1upgrade_pass.cpp

```diff
@@ -0,0 +1,48 @@
+//*****************************************************************************
```

Comment:
We should simplify this filename.

Reply:
Done, name changed to `product_opset_pass.cpp`